bootstrap.yml

```diff
@@ -2133,6 +2133,92 @@ packages:
       - args: ['mkdir', '-p', '@THIS_COLLECT_DIR@/usr/bin']
       - args: ['cp', '@THIS_BUILD_DIR@/sl', '@THIS_COLLECT_DIR@/usr/bin']
 
+  - name: libpipeline
+    default: false
+    source:
+      subdir: ports
+      url: 'http://download.savannah.gnu.org/releases/libpipeline/libpipeline-1.5.2.tar.gz'
+      format: 'tar.gz'
+      extract_path: 'libpipeline-1.5.2'
+      regenerate:
+        - args: ['cp',
+            '@BUILD_ROOT@/tools/host-automake-v1.15/share/automake-1.15/config.sub',
+            '@THIS_SOURCE_DIR@/build-aux/']
```

Comment:
This package needs a tool dependency on automake since it uses a file from automake. Likewise for the packages below.